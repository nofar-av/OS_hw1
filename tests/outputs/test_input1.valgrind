==28062== Memcheck, a memory error detector
==28062== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28062== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28062== Command: /home/omer/OS_wet1/smash
==28062== Parent PID: 28058
==28062== 
==28062== 
==28062== FILE DESCRIPTORS: 4 open at exit.
==28062== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_input1.valgrind
==28062==    <inherited from parent>
==28062== 
==28062== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_input1.err
==28062==    <inherited from parent>
==28062== 
==28062== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_input1.out
==28062==    <inherited from parent>
==28062== 
==28062== Open file descriptor 0:
==28062==    <inherited from parent>
==28062== 
==28062== 
==28062== HEAP SUMMARY:
==28062==     in use at exit: 0 bytes in 0 blocks
==28062==   total heap usage: 50 allocs, 50 frees, 83,492 bytes allocated
==28062== 
==28062== All heap blocks were freed -- no leaks are possible
==28062== 
==28062== For counts of detected and suppressed errors, rerun with: -v
==28062== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
