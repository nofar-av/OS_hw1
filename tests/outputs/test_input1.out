smash> smash> hello> hello> smash: got ctrl-Z
smash: process 28764 was stopped
hello> hello> 